Source/WebCore/cssjit/SelectorCompiler.cpp

```diff
@@ -446,6 +446,7 @@ struct SelectorFragment {
     Vector<SelectorList> matchesFilters;
     Vector<Vector<SelectorFragment>> anyFilters;
     const CSSSelector* pseudoElementSelector = nullptr;
+    Vector<const AtomString*> customStateSet;
```

Comment:
Please use `Vector<AtomString>` instead.

Reply:
This doesn't seem resolved.